src/lib/pickles/tag.mli

```diff
@@ -1,11 +1,18 @@
-open Core_kernel
```

Comment:
Revert

Reply:
This one, I am against reverting.

Interfaces should be as explicit as possible. They are, even more so than implementation code, for human consumption. One aspect of that is is to always fully qualify your types.

Reply:
Not approving without a revert, sorry :man_shrugging: